### PR TITLE
koord-scheduler: export client-go metrics

### PR DESCRIPTION
**Pull request by ZiMengSheng**
Signed-off-by: wangjianyu <zmsjianyu@gmail.com>

### Ⅰ. Describe what this PR does
this PR export client-go metrics to help diagnose analysis problems for koord-scheduler.

### Ⅱ. Does this pull request fix one issue?

fixes ##769

### Ⅲ. Describe how to verify it

image

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Review by eahydra — APPROVED** [review]
/lgtm

**Review by jasonliu747 — APPROVED** [review]
/lgtm

File: cmd/koord-scheduler/main.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/koordinator-sh/koordinator/pkg/scheduler/plugins/nodenumaresource"
 	"github.com/koordinator-sh/koordinator/pkg/scheduler/plugins/reservation"
 
+	_ "k8s.io/component-base/metrics/prometheus/clientgo"
```
Comment (jasonliu747) [inline cmd/koord-scheduler/main.go:37]: 像下面一样加个注释吧

Comment (ZiMengSheng) [reply cmd/koord-scheduler/main.go:38]: done

**Pull request merged**